app/assets/javascripts/models/beta/story.js

```diff
@@ -140,6 +140,15 @@ export const editStory = (story, newAttributes) => {
   newStory.estimate = isFeature(newStory) ? newStory.estimate : '';
   newStory.labels = Label.uniqueLabels(newStory.labels);
 
+  const handleState = (newStory) => {
```

Comment:
What do you think about moving this function to the outside of `editStory` (instead of creating it inside here) and giving it a name that says what it does? Maybe we could rename it to `stateFor(story)` and rename the argument to just `story` instead of `newStory`.

Reply:
Also, we need some tests for that :smile:

Reply:
There are already some tests, like when changing estimate, change state. Do you have any suggestions?? Yes, problably it's better to be outside.

Reply:
We need unit tests for the model since it's a new model rule.